pages/linux/speedread.md

```diff
@@ -0,0 +1,29 @@
+# speedread
+
+> A simple terminal-based open source Spritz-alike.
+> Shows input text as a per-word RSVP (rapid serial visual presentation) aligned on optimal reading points, which allows reading text at a much more rapid pace than usual as the eye can stay fixed on a single place.
+> More information: <https://github.com/pasky/speedread>.
+
+- Basic example.  Read `tea.txt` at a speed of 250 wpm:
+
+`cat tea.txt | speedread -wpm 250`
+
+- Resume from line:
+
+`cat tea.txt | speedread -resume 5`
+
+- Show multiple words at a time:
+
+`cat tea.txt | speedread -multiword`
+
+- Slow down by 10%:
+
+`[`
+
+- Speed up by 10%:
+
+`]`
+
+- Pause, and show the last few lines as context:
+
+`<space>`
```

Comment:
Hi, in addition to @sebastiaanspeck's comment I recommend to add these notes so the command examples wouldn't interfere with [`[`, the inbuilt Bash shell operator](https://nix.reinhart1010.id/en/common/%5B.html).

```suggestion
- Slow down by 10% during the reading session:

`[`

- Speed up by 10% during the reading session:

`]`

- Pause, and show the last few lines as context:

`<space>`
```

Reply:
> That seems a bit too unnecessarily verbose. How does this interfere with the inbuilt shell operator?

I agree with @reinhart1010's suggestion here, although an existing user of Speedread might know where this should be used, without a clear/verbose description it would confuse a lot of non-technical but casual Linux users when they execute it out of the blue, as to why the command is a Shell operator.

Reply:
It's specified that it's a keybinding, not an argument